### PR TITLE
Jacobi Iteration Method

### PR DESCRIPTION
**Pull request by eviltypha**
### **Describe your change:**

* [x] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by eviltypha** [comment]
Hi @cclauss, I've made the necessary changes for tests are failing. Please review it again

**Comment by A469L372K397** [comment]
No,in wile live to in Lin

________________________________
发件人: Nivas Manduva ***@***.***>
发送时间: 2021年10月20日 11:19
收件人: TheAlgorithms/Python ***@***.***>
抄送: Subscribed ***@***.***>
主题: Re: [TheAlgorithms/Python] Jacobi Iteration Method (#5113)


Hi @cclauss<https://github.com/cclauss>, the recent commit seems to be passing the workflows from my forked repo, do look into it.

―

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩

**Review by poyea — APPROVED** [review]

File: arithmetic_analysis/jacobi_iteration_method.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+        for j in range(0, cols - 1):
+            if i == j:
+                continue
+            else:
+                sum += table[i][j]
+
+        if table[i][i] <= sum:
+            raise ValueError("Coefficient matrix is not strictly diagonally dominant")
+
+    return is_diagonally_dominant
```
Comment (poyea) [inline arithmetic_analysis/jacobi_iteration_method.py:156]: We can throw in `jacobi_iteration_method` and always return a boolean here.

**Pull request merged**